pyiron/sphinx/base.py

```diff
@@ -587,10 +813,12 @@ def set_mixing_parameters(
 
     def set_occupancy_smearing(self, smearing=None, width=None):
         """
-        Set how the finite temperature smearing is applied in determining partial occupancies
+        Set how the finite temperature smearing is applied in
+        determining partial occupancies
 
         Args:
-            smearing (str): Type of smearing (only fermi si implemented anything else will be ignored)
+            smearing (str): Type of smearing (only fermi is
+                            implemented anything else will be ignored)
```

Comment:
I removed "(only fermi is supported in SPHInX)" from the error message below this, but I think we should keep the above message since only fermi is supported at the pyiron level right now. Adding MP-smearing, etc. should probably be the subject of another pull request. For now I think this is a good example of something that can be controlled by "expert" users via Groups e.g. `job.input.hamilton.MethfesselPaxton = 2`

Reply:
ok. I regard this fixed.